istqb-ct-genai-complete-guide.md

```
@@ -0,0 +1,1630 @@
+    CTFL["CTFL v4.0（前提資格：必須）"]
+
+    subgraph Specialist ["スペシャリスト資格 2025年NEW！"]
+        GenAI["CT-GenAI（Testing with Generative AI）<br/>生成AI・LLMを「使って」テストを行うための資格<br/>※本資格！"]
+        AI["CT-AI（2021年）<br/>AI・MLシステムを「テストする」ための資格<br/>※兄弟資格"]
+    end
+
+    CTFL --> GenAI
+    CTFL --> AI
+
+    GenAI --> Career["キャリアパス：<br/>CT-GenAI → CTAL-TA / CTAL-TTA / CTAL-TM → Expert Level"]
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)